### PR TITLE
unity-lite

### PR DESCRIPTION
**Pull request by bruno-garcia**
Sentry Unity Lite isn't a full Sentry SDK so lots of the current docs don't apply here.

How do I get the icons in?

They exist in the `platformicons`:
https://github.com/getsentry/platformicons/blob/master/svg/unity.svg
https://github.com/getsentry/platformicons/blob/master/svg_80x80/unity.svg

Seems we need to bump it

### CODE REVIEW

**Comment by bruno-garcia** [comment]
Build is broken:

>   platforms/unity/configuration/options#send-default-pii

Can't find a way to opt out of `configuration`

File: src/platforms/unity/index.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: Unity
```
Comment (PeloWriter) [inline src/platforms/unity/index.mdx:2]: Can you not adopt the common content here? https://docs.sentry.io/contributing/approach/write-getting-started/

**Review by PeloWriter — COMMENTED** [review]
It would be easier to adopt the common content for the Getting Started page, I've provided a link to those guidelines. But the Tl;dr is that you need only snap in the code samples, little to no prose required.

File: src/platforms/unity/index.mdx (hunk 1 above)
Comment (bruno-garcia) [reply src/platforms/unity/index.mdx:2]: I'll try. This SDK has nothing common with any other but I assume a "install, configure, test" I anyway added here so I can split it in 3 files.

**Pull request closed without merging**